### DISCUSSION TITLE
大一國文必要性

### DISCUSSION THREAD

**Discussion by cindy5D55** [1101-GS4719-C/Python-Community-GS4719-C#1515] [General]
想問一下大家對於大一國文的看法

大一國文應該是分成兩種，一種偏應用，一種偏古典文學，我自己是認為應用類的學到比較多，如email寫法或是論文寫法，古典文學就很看老師教得怎麼樣了。

不知道大家對於大一國文是否有不一樣的學習呢?

**Comment by brian431** [comment]
對於我讀的資工系來說我認為是完全沒有必要上國文課的。已經上了那麼多年的國文課真的不明白為什麼大學了還要將國文列入必修。

**Comment by poweichen00** [comment]
我覺得我上大一國文都是去聽故事居多，我認為可以增加應用類的內容。

**Comment by Chiuhsianglin** [comment]
我認為我修的大一國文最有收穫的就是企劃書，可以藉由跟不同系的人一起討論方案，從中學習，也會發現別組不同的創意。

**Comment by PaulSu0905** [comment]
我大一國文在上課時間讀了很多現代文學作品，我不覺得對我的未來特別有幫助，但就像讀有趣的小說一樣，沒有幫助但有些文章其實蠻有趣的，如果認真摸索的話說不定其中還蘊含著許多道理。

**Comment by Evening218** [comment]
我認為大一國文其實可以增加一些關於論文或其他有明確規範的類型寫作教學,以前已經學了很多古典或現代文學鑑賞的知識,至少必修課可以著重在更加實用的東西,我想文學素養應該比較適合放到通識課讓大家自由選擇

**Comment by MaxChi870** [comment]
我至今仍不明白大一國文的必修原因為何，不是說沒有任何收穫，而是跟本科系之間所學之內容幾乎無任何關聯性，尤其是對理工相關的科系而言。

**Comment by ThanatosJun** [comment]
我認為大一國文可分兩類，跟很樓上的一樣，一為應用類，二為古典類。前者的分析與文字轉化溝通對於團隊合作十分重要，後者則可以增加生活中的情趣與修養，培養EQ。

**Comment by lin157644** [comment]
坦白講我自己對於國文可謂一竅不通，應用類雖然也沒有擅長到哪裡，但是我覺得對於大學生活與未來都還算是有幫助，古典文學(我自己當時是有易經與節氣選讀兩種可選)，以資工來講，我就真的無法體會其用處，時至今日我甚至感覺一點收穫都沒有留下。

**Comment by 12Jesse24** [comment]
一上修了近代散文選讀，課程內容就跟國高中差不多，介紹某某作家的文章以及寫作時的心境，還有偷偷抱怨一下學校XD。
一下修了中文寫作，有教學如何寫企劃書也會需要你們分組報告，還有教你上研究所、轉系、面試所需要遞交的審查資料有那些雷點，甚至還有小面試主題包括轉系、申請工作、申請研究所...等等。課程內容就比較注重在團隊合作、學習新知、口語表達。

**Comment by THH1118** [comment]
我覺得大一國文如果是語文知識的話對我沒什麼幫助，但因為國文課很多的作業都需要分組，所以對我來講比較有用的地方就是訓練與別人合作吧。

**Comment by Chengbosco** [comment]
大一國文好像很看老師，我們劉心慧就很有趣，帶我們去看電影，玩戲曲，還有標點符號還有企畫書(有拿到錢，爽)，但其他人的國文課似乎就很痛苦。但我覺得國文課是必要的啦。

**Comment by eric202501** [comment]
其實大一國文沒有什麼作為必修的必要性吧，如果說要學習合作和企劃的話那有很多通識可以學到，但作為選修的話還不錯。

**Comment by davidteng00000** [comment]
我覺得大一國文完全沒有必要存在，我覺得我在高中已經獲得了良好充實的國語文訓練，大一國文對我而言只是一個學不到多少東西的負擔

**Comment by 111426027** [comment]
我覺得還是有存在的必要性，因為現今大多都是打字或是被網路用語影響，導致真的需要用到正式的詞語時，一時會想不到適當的詞彙，這時候就會有種書到用時方恨少的感覺。而且這應該也是人生中最後一次正式上國文課了，所以撐過去就好。

**Comment by Tomken123** [comment]
選一個涼的老師，整學期都不用去上課，分數還很甜

**Comment by mrweixd** [comment]
重點是要早起去上課，這對我來說實在太痛苦啦

**Comment by alvin119** [comment]
目前上到學期末了大一國文真的沒有太大必要。這學期上課重點主要在企劃書、讀書心得跟自傳，
在這其中對於自己的科系來講可能就只有自傳比較有幫助，但大一生實在沒什麼經歷好寫的。

**Comment by Terryyyyyyyyyyyy** [comment]
我認為甚至可以取消大一國文，我認為完全沒有必要，還不如學一些對未來有幫助的課程，如上述的email撰寫，就不用再叫我們去看一些經典文學，我們真的對那些東西不感興趣

**Comment by sagitariolotus** [comment]
我認為大一國文對理工科不是知識儲備方面的幫助，而是訓練人與人溝通的能力，課堂中有許多需要討論的議題

**Comment by chuanmin0317** [comment]
我覺得我在應用類的國文課上學到了蠻多東西的，像是寫email和企劃案寫作之類的，不過在古典文學中完全沒有學到任何的東西，畢竟其中的內容是現代社會中幾乎不會用到的文言文

**Comment by FakeWood** [comment]
我覺得學習有不能只看對自己之後找工作有沒有幫助，這種學習態度長期下來或許能讓人在某個領域專精，但我覺得也會讓這個人沒有深度，我說的深度不是指在一個領域裡鑽研的程度，而是我們跟一個人相處的時候，有時候會覺得這個人懂很多東西很有深度、有內涵，我覺得這些所謂的內涵就是一大堆看似對人生無所謂的知識，但其實他們或多或少都是你生命裡的養分

**Comment by TobywithDino** [comment]
我覺得大一下的國文就無聊到爆，過去聽沒用處的國文故事，讓我對大學教學很失望，以為可以擺脫國高中傳統的國語文教學。
但是我就很欣賞大一上的國文課，是要做活動規劃之類的，就比較偏向應用。

**Comment by lyykey** [comment]
大一國文真的沒有必要給理工科再修，就我而言，考大學已經因為爛國文而跑來中央讀資工，現在近來又告訴我要讀國文，真的是謝了。
但畢竟我也已經修完，且分數也都還行，但還是認為不應該給理工科修國文，因為真的沒有必要，謝謝。

**Comment by benbebobo149** [comment]
我蠻喜歡國文課中的應用文，就像樓上所提到的撰寫Email的禮儀。但我覺得學校在國文課中增加一個對於時事文章的批判，教導同學如何正確解讀這篇文章，避免我們超譯文章的內容。這是非常重要的一個能力，有很多媒體可能小時候沒有讀好國文課，對於撰寫一些實事，都會理解錯誤導致錯置概念。透過對時事文章的批判性閱讀，我們可以培養自己更深入、更全面的思考能力，並且提高溝通表達的準確性。我認為這樣的課程設計不僅有助於我們在學術上的成就，更能提升我們在現實生活中的適應能力。最後，我要感謝同學的分享，讓我有機會思考如何在國文課程中加強對時事文章的討論和批判性思維。

**Comment by 20050201Shawn** [comment]
我認為國文課還是有存在的價值，有時接觸理工東西太久就會想碰點語言類的東西放鬆心情，聽個古人八卦或看點文學作品，順便增加文學素養，當然是在國文老師沒給太多課業負擔的情況下啦。有些真的很不喜歡國文的，學校也可以提供抵修之類的課程，像是甚麼職場中文、理工科溝通課之類的(

**Comment by YANRONG0123** [comment]
我覺得大一國文偏文學性質的學分，是能夠培養我們抒發自己的情感，不讓別人覺得理工科系只是死板的表達一些邏輯或觀點，而寫作類型的話，就是在訓練表達能力，我認為這是相輔相成的。

**Comment by YifenLin** [comment]
對我來說，大一國文必修真的沒有甚麼必要性，只是為了要畢業，出現也只是為了點名，我自己常常在上課的時候寫微積分的作業(好孩子不要亂學)，不然就是跟國中的老朋友敘舊(好孩子不要亂學)。

**Comment by tzurou** [comment]
我自己覺得大一國文沒什麼必要性，基本的書信格式平常用不到，企劃書也是草草寫完，唯一比較實用的應該就是寫讀書心得吧，大一沒什麼機會可以寫長文，國文課就是可以訓練自己寫長文抒發自己想法的地方，但可能是我遇到的老師都蠻佛系的，所以我也就沒怎麼專心的上課順順的就過了。

**Comment by wanka2030** [comment]
國文素養的培養是在生活中會不斷更迭的，有人會因為外在的環境而受到影響，但大多數人會從自己的家庭中學會不同的習慣、說法。但大一國文的存在除了是讓自己能夠培養自身的國文能力不斷進步，也讓國文中的道理能夠不斷地複習，不單單是學會古人中發生的事，大多數都是希望我們能夠學會其中的道理!

**Comment by thomasxshen** [comment]
根本就不知道在幹嘛，感覺連老師自己也不清楚自己為什麼要教大一國文，明明是大學的課程卻搞得跟高四國文一樣無聊，超解⋯⋯

**Comment by happy246810** [comment]
我認為大一國文是非常必要的，因為大一國文不僅能培養大一新生的表達能力，幫助新生與其他人相處，同時也能增加大一新生的國文素養。

**Comment by EPHMR0524** [comment]
我覺得下學期的寫作有其必要性，不然很多人都可能不知道一些寫信的禮儀以及要注意的事情，但是如果說是文學鑑賞的部份我就認為不是非常的有必要，因為沒興趣的人根本就不會聽也學不到任何東西，而且重點是給分還都超爛，自己辛苦寫一篇超級長心得最後分數還不如別人找槍手代寫寫出來的，因此我覺得只要留中文寫作的部分就好了，文學鑑賞應該要改成通識課讓有興趣的人自己去學。

**Comment by William08290829** [comment]
大一國文偏應用的部分實用性還算可以，但古典的部分感覺就只是可能增加我們的文學素養而已，這就比較看個人的造化，有興趣的讀起來會覺得比較有趣。btw我就是沒興趣的那種。

**Comment by buoioiry** [comment]
大一國文選到有趣的老師上課還蠻開心的，但真的覺得有學到東西的應該只有書信，其他古文的部分都聽過就忘記了沒什麼實用性，有聽過朋友選到的老師考試比較難，對部分科系來說好像會是一個沒必要的負擔

**Comment by yyujimmy** [comment]
大一國文的教學方式和內容確實會因學校、老師以及課程設計而有所不同。一些大學可能更傾向於將國文課程設計為應用性較強的實用課程，例如教授寫作技巧、閱讀理解、文書溝通等，以幫助學生在未來的學習和職場生涯中更好地應用這些技能。

另一些大學可能更注重古典文學和文化的教育，著重於研讀古典文學作品、理解其背後的文化和歷史背景，以及培養學生的文學素養和人文精神。

無論是哪一種課程設計，都有其各自的價值和重要性。應用性強的課程能夠直接幫助學生提升實用技能，更好地應對日常生活和工作需求；而古典文學則能夠豐富學生的文化修養，培養其批判性思維和文學鑑賞能力。

最終，學生對於大一國文的看法可能會因個人興趣、學習目標和課程安排而有所不同。

**Comment by Mitochondrionjessica** [comment]
大一國文我覺得很看老師，有些老師非常用心的設計課程。像我上學期修國文寫作，不論是企劃書、採訪稿還是自傳，老師都有請業界的專家來課堂上，讓我們實際演練，雖然作業量多又繁重，但真的收穫滿滿。但下學期文學賞析，就像國高中國文課，讓我不經懷疑國語是母語的我又上了十二年的國文課，真的有必要修大一國文嗎。

**Comment by jim1743** [comment]
大一國文我認為比較實用的是應用性的國文課，因為能從中學到東西，然而像偏古典類的課，對非文組的我來說，沒有實際的幫助，只讓我覺得為何要學這種沒有意義的課。儘管我覺得偏應用的國文課很實用，但寧願希望大一不要有國文，因為國文真的不是我的興趣。
